pkg/ipmi/ocp/bootorder.go

```diff
@@ -23,13 +23,15 @@ const (
 	// 100b: Other removable Device
 	// If bit[2:0] is 001b (Network), bit3 determines IPv4/IPv6 order,
 	// when bit3 is 0: IPv4 first, bit3 is 1: IPv6 first
-	NETWORK_BOOT = 0x1
-	LOCAL_BOOT   = 0x2
-	INVALID_BOOT = 0xff
+	NETWORK_BOOT      = 0x1
+	NETWORK_BOOT_IPV6 = 0x1 | 0x8
+	LOCAL_BOOT        = 0x2
+	INVALID_BOOT      = 0xff
 
 	// Default boot order systembooter configurations
-	NETBOOTER_CONFIG   = "{\"type\":\"netboot\",\"method\":\"dhcpv6\"}"
-	LOCALBOOTER_CONFIG = "{\"type\":\"localboot\",\"method\":\"grub\"}"
+	NETBOOTER_CONFIG      = "{\"type\":\"pxeboot\"}" // IPV4 and IPV6 default to true
```

Comment:
For Facebook's use case I think it should be netboot (fbnetboot).
@insomniacslk is it okay to change from netboot to pxeboot for the default boot option? Although people can set their default booter configs using VPD at build time.

Reply:
I considered other ways to do this, but it seemed that the most generic way was best. The problem is VPD vs IPMI.

Reply:
Same as my comment in https://github.com/u-root/u-root/pull/2045#discussion_r716434905

Reply:
> Same as my comment in [#2045 (comment)](https://github.com/u-root/u-root/pull/2045#discussion_r716434905)

Seems reasonable if fbnetboot is needed. Can pxeboot in ipv6 mode be a full replacement?

Reply:
> > Same as my comment in [#2045 (comment)](https://github.com/u-root/u-root/pull/2045#discussion_r716434905)
> 
> Seems reasonable if fbnetboot is needed. Can pxeboot in ipv6 mode be a full replacement?

@insomniacslk what do you think?

Reply:
we need one, but I've never figured out how it should look. possibly the osf builder is the toold.